tests/subsys/cpp/libcxx/testcase.yaml

```diff
@@ -10,7 +10,7 @@ tests:
     tags: cpp
   cpp.libcxx.exceptions:
     # FIXME: change when issue is resolved #32448
```

Comment:
I think we can remove the FIXME comment at this point.  We can open issues to track sparc and NIOS support.

Reply:
Done